core/server/data/migrations/init/2-create-fixtures.js

```diff
@@ -10,6 +11,15 @@ module.exports = function insertFixtures(options) {
 
     return Promise.mapSeries(fixtures.models, function (model) {
         logging.info('Model: ' + model.name);
+
+        // The Post model fixtures need a `published_at` date, where at least the seconds
+        // are different, otherwise `prev_post` and `next_post` helpers won't workd with
+        // them.
+        if (model.name === 'Post') {
+            _.forEach(model.entries, function (post, index) {
+                post.published_at = moment().add(index, 'seconds');
```

Comment:
> If it becomes a problem for production sites I'd be inclined to change the date storage format so that we have millisecond resolution instead of resorting to adding seconds to published_at values here and there.

The default `dateTime` type can't store MS. This would be a big change.

---

@AileenCGN 
We have a fixture utility, which was written for fixtures. This allows us to write a JSON file for the initial blog data. Having a hack inside the migration script works, but imagine every fixture requirement adds a hack somewhere. So it would be great if this condition here get's moved to our fixture utility. 

Having dates in our fixtures was never a requirement, but as soon as we have a second use case, we should add support for it e.g. textual parsing `published_at: now() + ...` (can be done via a moment plugin for sure)